lib/stripe/topup.rb

```diff
@@ -2,17 +2,17 @@
 
 module Stripe
   class Topup < APIResource
-    extend Stripe::APIOperations::List
     extend Stripe::APIOperations::Create
+    extend Stripe::APIOperations::List
     include Stripe::APIOperations::Save
 
     OBJECT_NAME = "topup".freeze
 
     custom_method :cancel, http_verb: :post
 
-    def cancel
-      resp, api_key = request(:post, resource_url + "/cancel")
-      initialize_from(resp.data, api_key)
+    def cancel(params = {}, opts = {})
+      resp, opts = request(:post, resource_url + "/cancel", params, opts)
+      initialize_from(resp.data, opts)
```

Comment:
Codegen adds arguments to a method that used to have none. This is more in line with all other custom methods, but again, it's a public API change. Are we cool with it?

Reply:
Actually, `topup.cancel` does not accept any params: https://stripe.com/docs/api/topups/cancel

so while we should probably accept `opts`, I'm not sure we should accept `params`. @remi-stripe @ob-stripe any thoughts?

Reply:
Hmm, I think github ate a previous comment of mine saying that I think this is a peaceful change because it is backwards-compatible, which is still my bias..

Reply:
Yeah, I think this is peaceful. It doesn't break BC and adds the ability pass `opts`.

Even for methods that don't accept parameters, I think this should not be enforced in the clients. This way if the method is ever updated to accept parameters (and it has happened before: https://github.com/stripe/stripe-ruby/blob/master/CHANGELOG.md#300---2017-06-27) it's not a breaking change.